sympy/matrices/tests/test_commonmatrix.py

```diff
@@ -1268,3 +1271,52 @@ def test_jordan_form():
     P, J = m.jordan_form()
     assert all(isinstance(x, Float) or x == 0 for x in P)
     assert all(isinstance(x, Float) or x == 0 for x in J)
+
+def test_singular_values():
+    x = Symbol('x', real=True)
+
+    A = EigenOnlyMatrix([[0, 1*I], [2, 0]])
+    assert A.singular_values() == [2, 1]
+
+    A = eye(3)
+    A[1, 1] = x
+    A[2, 2] = 5
+    vals = A.singular_values()
+    assert 1 in vals and 5 in vals and abs(x) in vals
```

Comment:
I wonder why this old line was written in this form. Maybe `Abs` was not among the imports then.